### DISCUSSION TITLE
How do you integrate with design process and use same colors in your designs?

### DISCUSSION THREAD

**Discussion by romanr** [rydmike/flex_color_scheme#139] [Q&A] [answered]
Hi!
Trying to understand how to have Figma design use same colors as theme in FlexColor. 

- In Playground there's "Copy colour scheme" but it's not JSON or something that can be imported in Figma.
- Or maybe we can design using Google's Material 3 Figma library, if colors in Google's library would match theme in FlexColor?

Am  I missing  something obvious since this is a popular library and lot of projects use it?
What's everybody's design workflow?

**Comment by rydmike** [comment, marked as answer]
Hi @romanr, thanks for your question.

At the moment the **Themes Playground** is not a designer tool. It is main purpose is to be a convenient tool for developers to use as an alternative way to make an API configuration for the **FlexColorScheme** package. 

While it also allows for very flexible and extensive theme configuration, and to interactively try and experiment with different colors and styles on components, it can certainly also be used to test and try different design with, but it is currently not trying to be a designer tool. 

You can find some additional info on this here https://docs.flexcolorscheme.com/playground#future-plans  including future plans, where it is mentioned that some kind of Figma integration could be considered if Themes Playground is developed into a more "pro" app.

By the way, I understand that with the Figma M3 tool you can also generate M3 ColorSchemes. With the **FlexColorScheme** API, you can also use a predefined **ColorScheme** as an input ColorScheme, in a FlexColorScheme generated theme. You can't do this in the Themes Playground though.

**Reply by romanr** [reply]
> By the way, I understand that with the Figma M3 tool you can also generate M3 ColorSchemes. With the FlexColorScheme API, you can also use a predefined ColorScheme as an input ColorScheme, in a FlexColorScheme generated theme. You can't do this in the Themes Playground though.

That's the answer I was looking for! Will try and let you know.
